documentation/README.md

```diff
@@ -0,0 +1,23 @@
+Ensure that requirements in requirements_dev are full filled.
+
+Make changes to conf.py as per the requirements.
+
+To initialize the Sphinx source directory execute the following inside documentation:
+
+sphinx-quickstart
+
+Currently the directory has been initialized. 
+
+Use sphinx-apidoc to generate reStructuredText files from source code by executing following command:
+
+sphinx-apidoc -f -o <path-to-output> <path-to-module>
+
+Add the generated .rst file into index.rst toctree
+
```

Comment:
These steps are meant to guide the user through "re-creating" the conf.py and the .rst files that sphinx uses to build the final pdf/html files, correct?

@ta-oliver , was the idea of allowing users to re-build the documentation meant to also include these steps, or was it just meant to allow them to recreate the pdf/html based on the conf.py file and on the .rst files already provided in this repository? Because in the latter case, these steps can be cut out altogether.

Reply:
Generally the users won't recreate the documentation, but just access a PDF, so the information should be primarily aimed at allow project contributors to be able to update the documentation automatically before each release.